packages/api/src/types.ts

```diff
@@ -57,6 +57,14 @@ export interface IGraphQLRequest {
   variables?: IGraphQLVariables;
 }
 
+export interface IWebhookRequest {
+  method: string;
+  path?: string;
+  data?: {
```

Comment:
`data` option is the same as `query` option.

Reply:
How so? 

The goal of **_data_** is to be used on POST/PUT requests. On a GET and DELETE the Data prop would be ignored.

The goal of **_query_** is to easily let a user add query parameters to their request, for example:

```
let query = { search: "tacos" }
api.invoke("findRestaurants", { method: "GET", query })
=> This produces the path "$API_ENDPOINT/webhook/path?search=tacos"
```

Reply:
`event.data` in a webhook consist of parameters from query string in any case. I tested it today and I checked  this functionality in server-side code.